Source/ARTCrypto.h

```diff
@@ -28,13 +28,20 @@ NS_ASSUME_NONNULL_BEGIN
 @end
 
 /**
- ARTCipherParams contains configuration options for a channel cipher, including algorithm, mode, key length and key.
- Ably client libraries currently support AES with CBC, PKCS#7 with a default key length of 256 bits. All implementations also support AES128.
+ The ARTCipherParams object contains properties to configure encryption for a channel.
  */
 @interface ARTCipherParams : NSObject <ARTCipherParamsCompatible>
+
+/// The algorithm to use for encryption. Only AES is supported.
 @property (readonly, strong, nonatomic) NSString *algorithm;
+
+/// The private key used to encrypt and decrypt payloads.
 @property (readonly, strong, nonatomic) NSData *key;
+
+/// The length of the key in bits; for example 128 or 256.
 @property (readonly, nonatomic) NSUInteger keyLength;
+
+/// The cipher mode. Only "CBC" is supported.
```

Comment:
```suggestion
/// The cipher mode. Only `"CBC"` is supported.
```

@tbedford @m-hulbert as I mentioned elsewhere in this PR, I think this sort of formatting should be part of the canonical docs?

Reply:
This should just be `CBC` (no " ") as per the canonical table.

Reply:
https://github.com/ably/ably-cocoa/pull/1439/commits/0deb0a0ef9bd63bd007f7c36ffcac863575920a7